### PR TITLE
chore(stageTemplate): modify stageTemplate logic with resource type

### PR DESCRIPTION
**Pull request by huxiangtao**
modify stageTemplate logic with resource type select & style problem

**What this PR does / why we need it**:

add resource type select component & modify intl
image

**Which issue(s) this PR is related to** *(optional, link to 3rd issue(s))*:

Fixes #

Reference to #

**Special notes for your reviewer**:

/cc @caosm @qiuxiaolian @cd1989 

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: web/src/components/stageTemplate/add/ResourceType.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+// TODO huxt validate resource type select
+const ResourceType = props => {
+  const { setFieldValue, values, path, required } = props;
+  const resources = _.get(values, path, []);
+  return (
+    <FieldArray
+      name={path}
+      render={arrayHelpers => (
+        <Fragment>
```
Comment (caosm) [inline web/src/components/stageTemplate/add/ResourceType.js]: useless

File: web/src/locale/zh-CN.yaml [hunk 2]
```diff
@@ -120,6 +120,8 @@ template:
   deletetips: 确定删除模板？
   create: 创建模板
   update: 更新模板
+  resourceType: 资源类型
+  addRsourceType: 新增资源类型
```
Comment (cd1989) [inline web/src/locale/zh-CN.yaml]: Should be `Add Resource`, not `Add Resource Type`

**Comment by cd1989** [comment]
One more suggestion: display the deletion icon only when cursor hover on the list item.

File: web/src/locale/en-US.yaml [hunk 3]
```diff
@@ -120,6 +120,8 @@ template:
   deletetips: Are you sure to delete the template?
   create: Create
   update: Update
+  resourceType: Resource type
+  addRsourceType: Add resource type
```
Comment (cd1989) [inline web/src/locale/en-US.yaml]: `AddResource: Add resource`

**Pull request merged**